TranslationServices.Core/TranslationServiceFacade.cs

```diff
@@ -601,6 +601,25 @@ private static async Task<string[]> TranslateV3Async(string[] texts, string from
             }
         }
 
+        private static Boolean checkContains(string text, int min, int max)
+        {
+            return text.Where(e => e >= min && e <= max).Any();
```

Comment:
You can replace `Where` by `Any` here. Also general C# style is to use the lower case `bool` type (like int etc) even though they are technically aliases.

Reply:
Better name might be IsInUnicodeRange?